docs/upgrade.html

```diff
@@ -214,14 +220,41 @@ <h5><a id="upgrade_1020_streams" href="#upgrade_1020_streams">Upgrading a 0.10.1
     <li> See <a href="/{{version}}/documentation/streams#streams_api_changes_0102">Streams API changes in 0.10.2</a> for more details. </li>
 </ul>
 
+<h5><a id="upgrade_1020_streams_from_0100" href="#upgrade_1020_streams_from_0100">Upgrading a 0.10.0 Kafka Streams Application</a></h5>
+<ul>
+    <li> Upgrading your Streams application from 0.10.0 to 0.10.2 does require a <a href="#upgrade_10_1">broker upgrade</a> because a Kafka Streams 0.10.2 application can only connect to 0.10.2 or 0.10.1 brokers. </li>
+    <li> There are couple of API changes, that are not backward compatible (cf. <a href="/{{version}}/documentation/streams#streams_api_changes_0102">Streams API changes in 0.10.2</a> for more details).
```

Comment:
Could we just ref-link the `streams/upgrade-guide.html` here than copying the content again in the main upgrade section?

Reply:
We can. It's "break" the established pattern though. Up to now, those older sections are just the unchanged release notes from the version before -- I just followed the structure of the existing docs. Let me know what you think. Also, I am wondering, it this changes anyway if we rework the web page from scratch as anticipated.

Reply:
Sounds good. We can leave it as is then.